packages/salesforcedx-vscode-core/src/commands/forceDescribeMetadata.ts

```diff
@@ -28,18 +28,28 @@ export class ForceDescribeMetadataExecutor extends SfdxCommandletExecutor<
     return new SfdxCommandBuilder()
       .withArg('force:mdapi:describemetadata')
       .withJson()
-      .withLogName('force_describe_metadata')
+      .withLogName('force_mdapi_describemetadata')
       .build();
   }
 }
 
 export async function forceDescribeMetadata(
   outputFolder: string
 ): Promise<string> {
+  const startTime = process.hrtime();
+  const forceDescribeMetadataExecutor = new ForceDescribeMetadataExecutor();
   const execution = new CliCommandExecutor(
-    new ForceDescribeMetadataExecutor().build({}),
+    forceDescribeMetadataExecutor.build({}),
     { cwd: getRootWorkspacePath() }
   ).execute();
+
+  execution.processExitSubject.subscribe(() => {
```

Comment:
I don't believe this needs to be added since `execute()` isn't being overridden in the ForceDescribeMetadataExecutor, and the parent `execute()` takes care of logging the execution event:

in SfdxCommandletExecutor:
```
public execute(response: ContinueResponse<T>): void {
    const startTime = process.hrtime();
    const cancellationTokenSource = new vscode.CancellationTokenSource();
    const cancellationToken = cancellationTokenSource.token;
    const execution = new CliCommandExecutor(this.build(response.data), {
      cwd: getRootWorkspacePath()
    }).execute(cancellationToken);

    execution.processExitSubject.subscribe(() => {
      this.logMetric(execution.command.logName, startTime);
    });
    this.attachExecution(execution, cancellationTokenSource, cancellationToken);
}
```

Reply:
Oh you're not using the commandlet i see

Reply:
Yeah we can't use the SfdxCommandletExecutor when we need to collect info from the cli output, which means metrics aren't being reported. We've logged another bug to add this `logMetric` snippet to the other commands missing the telemetry as well.